pkg/operator/mon/mon.go

```diff
@@ -41,6 +41,10 @@ import (
 var logger = capnslog.NewPackageLogger("github.com/rook/rook", "op-mon")
 
 const (
+	// EndpointConfigMapName is the name of the configmap with mon endpoints
+	EndpointConfigMapName = "rook-ceph-mon-endpoints"
+	// Endpointkey is the name of the key inside the mon configmap to get the endpoints
+	DataKey           = "data"
```

Comment:
I'll update the DataKey name. @kokhang I can put the public consts in a separate const block if that makes it better